test/integration/tests/modules.cpp

```diff
@@ -54,6 +54,50 @@ TEST_F(ZNCTest, NotifyConnectModule) {
         "NOTICE nick :*** user@identifier detached from 127.0.0.1");
 }
 
+TEST_F(ZNCTest, ClientNotifyModule) {
+    auto znc = Run();
+    auto ircd = ConnectIRCd();
+    auto client = LoginClient();
+    client.Write("znc loadmod clientnotify");
+    client.ReadUntil("Loaded module");
+
+    auto check_not_sent = [](Socket& client, QString wrongAnswer){
+        auto result = client.ReadRemainder();
+        EXPECT_FALSE(result.contains(wrongAnswer.toUtf8())) << "Got an answer even though we didnt want one with the given configuration";
```

Comment:
Either add << result, or use EXPECT_THAT(result, Not(HasSubstr()) for a better error message

Reply:
✅